src/java.desktop/share/classes/javax/swing/plaf/synth/SynthLookAndFeel.java

```diff
@@ -615,6 +615,16 @@ public void load(InputStream input, Class<?> resourceBase) throws
      * <code>new URL(synthFile, path)</code>. Refer to
      * <a href="doc-files/synthFileFormat.html">Synth File Format</a> for more
      * information.
+     * <p>
+     * Whilst this API may be safe for loading local resources that are
+     * delivered with a {@code LookAndFeel} or application, and so have an
+     * equal level of trust with application code, using it to load from
+     * from remote resources, particularly any which may have a lower level of
+     * trust, is strongly discouraged.
+     * The alternative mechanisms to load styles from an {@code InputStream}
+     * using a resource co-located with the application or by
+     * providing a {@code SynthStyleFactory} are preferred.
+     * Consequently this method is deprecated and will be removed in a future release.
```

Comment:
```suggestion
     * Consequently, this method is deprecated and will be removed in a future release.
```
Shall the comma be added?